From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthew Jeffryes <mjeffryes@pulumi.com>
Date: Mon, 6 May 2024 16:50:17 -0700
Subject: [PATCH] Add shim


diff --git a/sdwan/shim.go b/sdwan/shim.go
new file mode 100644
index 00000000..29a8db3d
--- /dev/null
+++ b/sdwan/shim.go
@@ -0,0 +1,12 @@
+package sdwan
+
+import (
+	sdwan "github.com/CiscoDevNet/terraform-provider-sdwan/internal/provider"
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+)
+
+const version = "dev"
+
+func Provider() provider.Provider {
+	return sdwan.New(version)()
+}
